python/roadgraphtool/db.py

```diff
@@ -140,15 +140,17 @@ def get_new_psycopg2_connection(self):
             return None
 
     @connect_db_if_required
-    def execute_sql(self, query, *args, use_transactions=True) -> None:
+    def execute_sql(self, query, *args, schema='public', use_transactions=True) -> None:
         """
         Execute SQL that doesn't return any value.
         """
         with self._sqlalchemy_engine.connect() as connection:
             if not use_transactions:
                 connection.execution_options(isolation_level="AUTOCOMMIT")
             with connection.begin():
+                connection.execute(sqlalchemy.text(f"SET search_path TO {schema};"))
                 connection.execute(sqlalchemy.text(query), *args)
+                connection.execute(sqlalchemy.text(f"SET search_path TO public;"))
```

Comment:
this does not seem right :)

Reply:
You can save starting value of search_path by executing ‘search_path_tmp = db.execute_sql_and_fetch_all_rows(“show search_path;”)[0][0]’

Reply:
no, I mean the path hardcoded to the public scheme? Is that correct?